app/views/docs/database.phtml

```diff
@@ -0,0 +1,216 @@
+</div>
+
+
+<h2><a href="/docs/advanced-database#attributes" id="attributes">Create Attributes</a></h2>
+<p>
+    Once you choose your permission model, navigate to your collection's <b>Attributes</b> tab. Attributes are used to define the structure of your documents and help the Appwrite API validate your users' input. Add your first attribute by clicking the <b>Add Attribute</b> button. You can choose between the following types:
+</p>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 250px">Attribute</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>string</td>
+            <td>String attribute.</td>
+        </tr>
+        <tr>
+            <td>integer</td>
+            <td>Integer attribute.</td>
+        </tr>
+        <tr>
+            <td>float</td>
+            <td>Float attribute.</td>
+        </tr>
+        <tr>
+            <td>boolean</td>
+            <td>Boolean attribute.</td>
+        </tr>
+        <tr>
+            <td>enum</td>
+            <td>Enum attribute.</td>
+        </tr>
+        <tr>
+            <td>ip</td>
+            <td>IP address attribute for IPv4 and IPv6.</td>
+        </tr>
+        <tr>
+            <td>email</td>
+            <td>Email address attribute.</td>
+        </tr>
+        <tr>
+            <td>url</td>
+            <td>URL attribute.</td>
+        </tr>
+    </tbody>
+</table>
+
+<p>
+    If an attribute must be present in all documents, set it as <span class="tag">required</span>. If not, a <span class="tag">default</span> value might be handy. Additionally, decide if the attribute should be a primitive or array of values.
+</p>
+<p>
+    When adding or removing attributes, your requests are processed in the background, indicated by the attribute's <span class="tag">status</span>.
+    Depending on your collection's size and other factors, this could take anywhere from a few seconds to a few minutes to complete. Once all of your attributes are <span class="tag">available</span>, then you can create a document.
+</p>
+
+<h3><a href="/docs/advanced-database#create-documents" id="create-documents">Create Documents</a></h3>
+<p>
+    Navigate to the <b>Documents</b> tab of your collection and click the <b>Add Document</b> button, or add a document programmatically:
+</p>
+<div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+    <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Appwrite } from "appwrite";
+
+const sdk = new Appwrite();
+const promise = sdk.database.createDocument('[COLLECTION_ID]', {});
+
+promise.then(function (response) {
+    console.log(response); // Success
+}, function (error) {
+    console.log(error); // Failure
+});
+</code></pre>
+</div>
+
+<h3><a href="/docs/advanced-database#finding-documents" id="finding-documents">Finding Documents</a></h3>
+<p>
```

Comment:
```suggestion

<div class="notice margin-top-large margin-bottom-large">
    <p>Note: You can only query correctly indexed documents. More details on this can be found <a href="/docs/advanced-database#indexes" id="indexes">here.</a> </p>
</div>
<p>
```